### DISCUSSION TITLE
Stuck on downloading files.

### DISCUSSION THREAD

**Discussion by siddharth-movaliya** [vercel/turborepo#6860] [Help] [answered]
So I'm using a MacBook Air M1 and tried running:

    npx create-turbo@latest

It then asks to give a name for the project and the option to select workspace type like rpm, yarn, bun, etc.

The problem I'm facing is that once I do that it starts downloading the files step but gets stuck on this message:

    Downloading files. This might take a moment.

Does anybody have any solutions?

**Comment by anthonyshew** [comment, marked as answer]
Duplicate of #6861
